### ISSUE TITLE
Beta test with PAWS GMC catalog - Comments and Suggestions

### ISSUE THREAD

**Issue by dcolombo** [camelot-project/frontend#144]
- Issues:

Generally, I did not have problem to understand how to upload the data, and to actually upload them. The instructions are pretty clear and straightforward. However, I was surprised when my data did not show up together with the others, by using the query tool.

The table created with the uploaded data appeared cut on one side (i.e. it overflowed on the right side of the page, above the background, and it was not visible).
- Features (sparse ideas and wish list):

1) I would like to have, maybe together with the uploader name also:
- name of the galaxy/data origin for the uploaded dataset,
- tracer,
  -...;

2) a way to visually differentiate between two datasets uploaded by the same user;

3) possibility to remove personally uploaded datasets;

4) on the table that appears passing the pointer over the data points, a way to fix the table in order, for example, to retrieve or to connect directly to the cited data paper;

5) it would be nice to have several interactive toggle buttons, that allow to highlight, i.e. a specific tracer, Galactic/extragalactic data, simulations/observations, and, for example, color in grey the other data points. I would prefer this instead to completely remove data from the common plot. This tool might be useful to find possible biases, environmental effects, etc.  introduced, for instance, by a given tracer;

6) some other plots, as the classic size-line width relation, scaling parameter vs the mass spectra. The latter might be useful for comparison with simulations;

7) some classic relations to guide the eyes, like the Solomon et al. 1987' size - line width relation for  GMCs;

8) some basic statistics, like a linear fit for the data points, or a power-law fit for the mass spectra.

**Comment by keflavich** [comment]
@dcolombo - thanks!

A few replies:
- Your data will not show up until one of the camelot members merges it.  This is the curation step, and it is intentional.
- The table display issue and difficulty differentiating data sets are real problems.  We have an open set of issues for the plot overlap problems (e.g. #123), but the table display is new.

Your issue 3 can be done, but we don't plan on automating it - if you want a dataset removed, you have to issue a pull request to remove it.

I don't understand your point 4, could you rephrase?

Point 5 has an open issue #118.

Point 6 is something we may not try to do - you can easily do this by downloading the data, though.

Point 7 is a reasonable idea.  Maybe we could add some classical classification overlays.  E.g., core-clump-cloud-molecular association-galaxy-galaxy cluster.

I don't know what you want for point 8.  I hope the data do not fall along a line!  Similarly, I hope the combinations of simulations and observations do not follow a mass spectrum.  You could ask for these for any given data set, sure, but that's at present not our intent - analysis should happen elsewhere (e.g., with turbustat), camelot just hosts & presents the metadata.

**Comment by dcolombo** [comment]
Thanks for your replies!

About point 4, I noted that when passing the pointer over the data points in the plot, a table with several useful information (as uploader name, object ID, ADS name, DOI etc.) appears. It would be nice to have a way to keep this table visible and to navigate it. I would like to be able to go on the DOI tag and open directly the related data paper.

For point 8, I meant for each given dataset. However, I agree with you that this is not something very important to have at the moment.

**Comment by scog1234** [comment]
Quoting Adam Ginsburg notifications@github.com:

> @dcolombo - thanks!
> 
> A few replies:
> - Your data will not show up until one of the camelot members merges  
>   it.  This is the curation step, and it is intentional.

  We should make sure that we explain this properly in the
  documentation, and possibly also in the immediate feedback that
  you get after submitting a dataset.

  Simon
